modules/node/src/swapRate/swapRate.service.ts

```diff
@@ -69,9 +69,12 @@ export class SwapRateService implements OnModuleInit {
       newRate = await Promise.race([
         new Promise(async (resolve, reject): Promise<void> => {
           switch (priceOracleType) {
-            case "UNISWAP":
+            case PriceOracleTypes.UNISWAP:
               resolve(await this.getUniswapRate(from, to));
               break;
+            case PriceOracleTypes.HARDCODED:
+              resolve(await this.config.getDefaultSwapRate(from, to));
```

Comment:
Good for now, but I was thinking we could have a way for a custom hardcoded rate to be provided via the config, within the `allowedSwap` env var.

Reply:
The `AllowedSwap` type can't support this w/out some changes. I agree that being able to specify a swap rate as part of env vars would be good but don't think it's worth the effort at this point.